cheroot/server.py

```diff
@@ -1406,10 +1393,7 @@ def get_peer_creds(self):  # LRU cached on per-instance basis, see __init__
             https://github.com/daveti/tcpSockHack
             msdn.microsoft.com/en-us/commandline/wsl/release_notes#build-15025
             """
-            six.raise_from(  # 3.6+: raise RuntimeError from socket_err
-                RuntimeError,
-                socket_err,
-            )
+            raise RuntimeError() from socket_err
```

Comment:
You don't need `()` if there's no args:
```suggestion
            raise RuntimeError from socket_err
```

Reply:
I thought the preferred usage was to always raise an Exception instance instead of Exception class. I'm a little surprised the linters haven't taken a stance on the equivalent syntaxes.

Reply:
The interpreter initializes these automatically so it'll become an instance regardless. I think it's a well-documented feature.